opal/mca/common/ofi/common_ofi.c

```diff
@@ -482,10 +482,16 @@ static int compute_dev_distances(pmix_device_distance_t **distances,
     if (PMIX_SUCCESS != ret) {
         goto out;
     }
+    /* if we are not bound, then we cannot compute distances */
+    if (hwloc_bitmap_iszero(cpuset.bitmap) ||
```

Comment:
is cpuset.bitmap suppose to be an input parameter? It doesn't seem to be set to anything before we use it here. Should it be initialized before hand?

Reply:
nevermind.... sorry, the code with collapsed in the github view... change looks good.